e2e/assert.go

```diff
@@ -154,3 +157,59 @@ func testPodsWithPredicate(pods []corev1.Pod, pred func(*corev1.Pod) bool) bool
 func statefulSetIsReady(ss *appsv1.StatefulSet) bool {
 	return ss.Status.ReadyReplicas == ss.Status.Replicas
 }
+
+func requireDatabaseToFunction(t *testing.T, sb testenv.DiffingSandbox, b testutil.ClusterBuilder) {
+	sb.Mgr.GetConfig()
+	podName := fmt.Sprintf("%s-0.%s", b.Cluster().Name(), b.Cluster().Name())
```

Comment:
This is failing because of SSL certificate issues.  In order for the port forwarder code to work it has to look up a pod name, but the pod name such as crdb-0 is not a host in the SSL certificates.